vscode/src/telemetry.ts

```diff
@@ -43,11 +43,7 @@ export function initTelemetry(context: vscode.ExtensionContext) {
   }
   // see issue here: https://github.com/microsoft/vscode-extension-telemetry/issues/185
   // we cannot use the latest version of extension-telemetry until this is fixed
-  reporter = new TelemetryReporter(
-    "qsharp-vscode",
-    packageJson.version,
-    packageJson.aiKey
-  );
+  reporter = new TelemetryReporter(packageJson.aiKey);
```

Comment:
I assume the extension id and version are still captured in the telemetry somehow by the library?